operators/src/plot/temporal_raster_mean_plot.rs

```diff
@@ -0,0 +1,438 @@
+use crate::engine::{
+    ExecutionContext, InitializedOperator, InitializedOperatorImpl, InitializedPlotOperator,
+    Operator, PlotOperator, PlotQueryProcessor, PlotResultDescriptor, QueryContext, QueryProcessor,
+    QueryRectangle, RasterQueryProcessor, TypedPlotQueryProcessor,
+};
+use crate::error;
+use crate::util::math::average_floor;
+use crate::util::Result;
+use async_trait::async_trait;
+use futures::stream::BoxStream;
+use futures::StreamExt;
+use geoengine_datatypes::plots::{AreaLineChart, Plot, PlotData};
+use geoengine_datatypes::primitives::{Measurement, TimeInstance, TimeInterval};
+use geoengine_datatypes::raster::{Pixel, RasterTile2D};
+use serde::{Deserialize, Serialize};
+use snafu::ensure;
+use std::collections::BTreeMap;
+
+pub const TEMPORAL_RASTER_MEAN_PLOT_NAME: &str = "Temporal Area Mean Plot";
```

Comment:
I don't really like the name
- Why did you decide to use an area plot here in the first place? I think it could be optional to fill the area under the line
- The operator only works for rasters, do we want to extend it in the future to handle feature collections? Then we have the Option<FeatureAttribute> again. Otherwise the word raster should be mentioned so users can find it better. Also: do we want to call every plot XyZPlot? Clippy would be furious. ;) I would suggest "Mean Raster Pixel Values over Time"

Reply:
> I don't really like the name
> 
>     * Why did you decide to use an area plot here in the first place? I think it could be optional to fill the area under the line
Because it is the demand of a use case.

>     * The operator only works for rasters, do we want to extend it in the future to handle feature collections? Then we have the Option again. Otherwise the word raster should be mentioned so users can find it better. Also: do we want to call every plot XyZPlot? Clippy would be furious. ;) I would suggest "Mean Raster Pixel Values over Time"
Yes, I would like to have something for features, but this is much more complicated, since each feature has its own validity. We have to specify buckets/steps there.

>     * The disctinction between name and plot is still a bit fuzzy imho. What do we show to the user und how do we distinguish plot types?
I guess we can use a user friendly name for that.